src/test/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMSourceTraitsTest/repositoryUrl.xml

```diff
@@ -1,5 +1,6 @@
 <org.jenkinsci.plugins.github_branch_source.GitHubSCMSource>
   <id>e4d8c11a-0d24-472f-b86b-4b017c160e9a</id>
+  <apiUri>https://api.github.com</apiUri>
```

Comment:
In case other reviewers are curious about this in terms of compatibility: The previous version of the file did not represent a configuration state that could be reached from the UI. After #236, `apiUri` is guaranteed non-null, and this config specifies `repositoryUrl`, which means it requires #236.

It would be possible to have a null `apiUri` if `repositoryUrl` was null, and that would represent the pre-236 to post-236 upgrade for all instances of `GitHubSCMSource`, and in that case, `setApiUri` would be called in `readResolve` to make `apiUri` nonnull, and that call would be respected rather than ignored because `repositoryUrl` would be null.

Reply:
I'm going to un-resolve this comment, so that reviewers can easily see it. Otherwise I think GitHub tends to hide it.